cmd/maestro/server/grpc_broker.go

```diff
@@ -277,7 +282,7 @@ func (s *GRPCBroker) PredicateEvent(ctx context.Context, eventID string) (bool,
 	if svcErr != nil {
 		// if the resource is not found, it indicates the resource has been handled by other instances.
 		if svcErr.Is404() {
-			log.Debugf("The resource %s has been deleted, mark the event as reconciled", evt.SourceID)
+			logger.V(4).Info("The resource has been deleted, mark the event as reconciled", "source", evt.SourceID)
```

Comment:
```suggestion
			logger.V(4).Info("The resource has been deleted, mark the event as reconciled", "resourceID", evt.SourceID)
```

here the event source id should be resource id

Reply:
evt.SourceID should this be also changed to resourceID?